slices/openssh-client.yaml

```diff
@@ -0,0 +1,49 @@
+package: openssh-client
+
+essential:
+  - openssh-client_copyright
+
+slices:
+  bins:
+    essential:
```

Comment:
Passwd is a bit of a special package, because most of the contents are only used by libpam, or user management. I added the package + deps - with comments explaining each slice. I also added a dependency on passwd here